includes/wikia/services/VideoService.class.php

```diff
@@ -49,9 +52,23 @@ public function addVideo( $url ) {
 				$file = wfFindFile( $videoTitle );
 			}
 
-			// Add a default description if available and one doesn't already exist
-			$vHelper = new VideoHandlerHelper();
-			$vHelper->addDefaultVideoDescription( $file );
+			if ( $file instanceof File )  {
```

Comment:
I might be a n00b, so I'll ask: what is wrong with file being a file object, why there is an error thrown?